From: Takashi Iwai <tiwai@suse.de>
Subject: kABI workaround for pci quirks
Patch-mainline: Never, kABI workaround
References: git-fixes

Containing a kABI workaround due to drivers/pci/quirks.c changes.

patches.suse/PCI-Work-around-Intel-I210-ROM-BAR-overlap-defect.patch
adds a new bit field in struct pci_dev.  Hide with the standard
ifdef.

Signed-off-by: Takashi Iwai <tiwai@suse.de>

---
 include/linux/pci.h |    2 ++
 1 file changed, 2 insertions(+)

--- a/include/linux/pci.h
+++ b/include/linux/pci.h
@@ -457,7 +457,9 @@ struct pci_dev {
 	unsigned int	link_active_reporting:1;/* Device capable of reporting link active */
 	unsigned int	no_vf_scan:1;		/* Don't scan for VFs after IOV enablement */
 	unsigned int	no_command_memory:1;	/* No PCI_COMMAND_MEMORY */
+#ifndef __GENKSYMS__
 	unsigned int	rom_bar_overlap:1;	/* ROM BAR disable broken */
+#endif
 	pci_dev_flags_t dev_flags;
 	atomic_t	enable_cnt;	/* pci_enable_device has been called */
 
